test/language/default-parameters/replace-default-values.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.2.1
+description: >
+  Replace default values
```

Comment:
This could use more formal justification, e.g.

    > 24. If hasDuplicates is true, then
    >     [...]
    > 25. Else,
    >     a. Let formalStatus be IteratorBindingInitialization for formals with
    >        iteratorRecord and env as arguments.

    > ES6 13.3.3.6 Runtime Semantics: IteratorBindingInitialization
    >
    > SingleNameBinding : BindingIdentifier Initializeropt
    >
    > [...]
    > 6. If Initializer is present and v is undefined, then
    >    [...]

Which suggests a good extension: why not test specifying all the falsey
    values as arguments (to be sure the implementor isn't doing a type coercion
    when applying defaults)